### ISSUE TITLE
µg UnifiedNlp get's confused with Google Play Services

### ISSUE THREAD

**Issue by utack** [JusticeRage/ApkTrack#24]
As it seems the F-Droid app "µg UnifiedNlp", a network location provicer for gapps free devices, gets confused with the original Google Play Services (currently version 6.7.76 1745988-036) in the version check.
![screenshot_2015-03-13-16-12-49](https://cloud.githubusercontent.com/assets/8551234/6641125/483f4686-c99c-11e4-9b14-36bdbb6baa22.png)

**Comment by JusticeRage** [comment]
Hi, and thank you very much for reporting this.
From what I understand about µg UnifiedNlp, it is a drop-in (foss) replacement for google apps. And by replacement, I mean this:

```
$> aapt dump badging Android/NetworkLocation.apk | grep package:\ name
package: name='com.google.android.gms' versionCode='1300' versionName='1.3.0'
```

As shown above, it uses the exact same package name as Google Play Services (`com.google.android.gms`). This is of course a problem for ApkTrack, since this is the information it uses to identify an application - [package names are supposed to be unique](https://developer.android.com/guide/topics/manifest/manifest-element.html#package), after all. µg UnifiedNlp does its best to present itself as a Gapp for the rest of the system, so it's no wonder it fools ApkTrack as well.

Although ApkTrack will never be able to tell the difference, there is an upcoming feature (#4) that will allow users to select from which website they want to get update information. F-Droid will be included and selecting it will cause this package name to point to the [following page](https://f-droid.org/repository/browse/?fdfilter=com.google.android.gms&fdid=com.google.android.gms), which is the one you want.

In other words, this should get fixed in the near future (#4 is what I'm currently working on).
